summarization/text_summarization.py

```diff
@@ -0,0 +1,54 @@
+''' Module to perform text summarization'''
+
```

Comment:
This file was deleted in a previous commit.
Restoring it here, Feel free to not review this!